src/analytics.ts

```diff
@@ -58,6 +66,8 @@ export function createAnalyticsInstance(options?: Options) {
         getId,
         trackEvent,
         getInstances,
+        pageView,
+        setAccountType,
```

Comment:
why do we still need this function?
in `setAttributes` we already get the `account_type`